src/main/java/br/ufpe/cin/mergers/SemistructuredMerge.java

```diff
@@ -37,33 +37,38 @@
  * Represents semistructured merge. Semistrucutred merge is based on the concept
  * of <i>superimposition</i> of ASTs. Superimposition merges trees recursively,
  * beginning from the root, based on structural and nominal similarities.
+ * 
  * @author Guilherme
  */
 public final class SemistructuredMerge {
 
+	private enum SuperimpositionStep {
+		Left_Base, LeftBase_Right
+	}
+
 	static final String MERGE_SEPARATOR = "##FSTMerge##";
 	static final String SEMANTIC_MERGE_MARKER = "~~FSTMerge~~";
 
 	private static List<ConflictHandler> assembleListOfHandlers() {
 		ImmutableList.Builder<ConflictHandler> builder = new ImmutableList.Builder<>();
 
-		if(JFSTMerge.isTypeAmbiguityErrorHandlerEnabled)
+		if (JFSTMerge.isTypeAmbiguityErrorHandlerEnabled)
 			builder.add(new TypeAmbiguityErrorHandler());
 
-		if(JFSTMerge.isNewElementReferencingEditedOneHandlerEnabled)
+		if (JFSTMerge.isNewElementReferencingEditedOneHandlerEnabled)
 			builder.add(new NewElementReferencingEditedOneHandler());
 
-		if(JFSTMerge.isMethodAndConstructorRenamingAndDeletionHandlerEnabled &&
-			!JFSTMerge.isLegacyMethodAndConstructorRenamingAndDeletionHandlerEnabled)
+		if (JFSTMerge.isMethodAndConstructorRenamingAndDeletionHandlerEnabled
+				&& !JFSTMerge.isLegacyMethodAndConstructorRenamingAndDeletionHandlerEnabled)
 			builder.add(new MethodAndConstructorRenamingAndDeletionHandler());
 
-		if(JFSTMerge.isLegacyMethodAndConstructorRenamingAndDeletionHandlerEnabled)
+		if (JFSTMerge.isLegacyMethodAndConstructorRenamingAndDeletionHandlerEnabled)
```

Comment:
@jvcoutinho can we start using the new names for the renaming handlers?

Reply:
Yes, I'm working on this.

Reply:
I'll probably send another PR for this.